taipy/gui/data/array_dict_data_accessor.py

```diff
@@ -26,8 +25,8 @@ class _ArrayDictDataAccessor(_PandasDataAccessor):
     def get_supported_classes() -> t.List[str]:
         return [t.__name__ for t in _ArrayDictDataAccessor.__types]  # type: ignore
 
-    def _get_dataframe(self, value: t.Any) -> t.Union[t.List[pd.DataFrame], pd.DataFrame]:
-        if isinstance(value, list):
+    def to_pandas(self, value: t.Any) -> t.Union[t.List[pd.DataFrame], pd.DataFrame]:
+        if isinstance(value, (list, tuple)):
```

Comment:
`isinstance(value, Iterable)`? I'm thinking numpy arrays...

Reply:
numpy are dealt with in numpy_data_accessor :-)